src/runtime/virtcontainers/qemu.go

```diff
@@ -460,6 +461,42 @@ func (q *qemu) setupFileBackedMem(knobs *govmmQemu.Knobs, memory *govmmQemu.Memo
 	memory.Path = target
 }
 
+func (q *qemu) createVirtiofsd() (Virtiofsd, error) {
```

Comment:
`createVirtiofsd` and `virtiofsdSocketPath` are some bit confused, suggest changing to more meaningful names.